genestack_client/file_types.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+
+class JavaFileTypes(object):
+
+    @classmethod
+    def is_file_type(cls, type_str):
```

Comment:
How about moving this to utils:

```python
def validate_constant(cls, type_str):
    types = {v for k, v in cls.__dict__.iteritems() if (not k.startswith("_") and isinstance(v, basestring))}
    return type_str in types
```

This class should be clear from all methods, because you document all undoc member for it.